tests/integration/ha_tests/test_self_healing.py

```diff
@@ -79,10 +79,16 @@ async def test_build_and_deploy(ops_test: OpsTest) -> None:
 
 
 @pytest.mark.group("ha_tests")
+@pytest.mark.abort_on_fail
 @pytest.mark.parametrize("process", DB_PROCESSES)
-@pytest.mark.parametrize("signal", ["SIGTERM", pytest.param("SIGKILL", marks=markers.juju2)])
+@pytest.mark.parametrize("signal", ["SIGTERM", "SIGKILL"])
 async def test_interruption_db_process(
-    ops_test: OpsTest, process: str, signal: str, continuous_writes, primary_start_timeout
+    ops_test: OpsTest,
+    process: str,
+    signal: str,
+    continuous_writes,
+    primary_start_timeout,
+    restart_policy,
```

Comment:
Pebble's default backoff is too low (half a second) for the Patroni kill test to work. The restart policy bumps that significantly (for full cluster restart), but we have to bump other timeouts.